src/server_state.cpp

```diff
@@ -1285,20 +1304,21 @@ void wsrep::server_state::close_orphaned_sr_transactions(
         }
     }
 
-
     streaming_appliers_map::iterator i(streaming_appliers_.begin());
     while (i != streaming_appliers_.end())
     {
-        if (std::find_if(current_view_.members().begin(),
-                         current_view_.members().end(),
-                         server_id_cmp(i->first.first)) ==
-            current_view_.members().end())
+        bool origin_not_in_view = std::find_if(current_view_.members().begin(),
```

Comment:
`origin_not_in_view` should be const.
Also if `equal_consecutive_views` is true, there is no need to evaluate it ``origin_not_in_view`, consider evaluating it only if `equal_consecutive_view` is false

Reply:
removed the tmp var (added a small comment) to use short-circuiting here